src/ports/rt-kernel/sampleapp_main.c

```diff
@@ -185,7 +185,7 @@ int main (void)
       os_timer_create (APP_TICK_INTERVAL_US, main_timer_tick, NULL, false);
    gp_appdata = &appdata;
 
-   printf ("\n** Profinet sample application **\n");
+   printf ("\n** Starting Profinet demo application " PNET_VERSION " **\n");
```

Comment:
Should we settle on either _sample_ or _demo_? Currently we seem to be using both.

Reply:
Changed to sample app to relevant places in the code and documentation.
Done.